db/dcrsqlite/apisource.go

```diff
@@ -1098,7 +1098,12 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		}
 	}
 	numberMaxOfTx := int64(len(txs))
-
+	var numFundingTxns int64
+	if numberMaxOfTx < count {
+		numFundingTxns = numberMaxOfTx
+	} else {
```

Comment:
This should do it.  But note that more idiomatic Go would be to avoid the `else` by initializing with `numFundingTxns := count` rather than zero initializing.